src/Domain/Domain.hpp

```diff
@@ -114,11 +119,18 @@ class Domain {
     return blocks_;
   }
 
+  const std::unordered_map<std::string, ExcisionSphere<VolumeDim>>
+  excision_spheres() const noexcept {
```

Comment:
Suggestion: How about we store this map of excision spheres in a `domain::Tags::ExcisionSpheres` tag in the DataBox, instead of nesting it in the `Domain` class? Any pros / cons for this?

Reply:
summary of discussion with @nilsdeppe - We're unsure yet precisely how we might use the ExcisionSpheres, so we will go with this for now